backend/file_io.py

```diff
@@ -39,7 +39,7 @@ def log(*msg):
         tstamp = datetime.datetime.now()
         try:
             logfile = None
-            logfile = open('logfile.txt',"a")
+            logfile = open('otone_data/logfile.txt',"a")
```

Comment:
we'll need to do some testing as an executables, but the otone_data folder is created by the code to store machine-specific data, so I figured the logfile should also go in there